docs/reference/data-streams/downsampling-ilm.asciidoc

```diff
@@ -37,8 +37,16 @@ To enable downsampling, add a <<ilm-downsample,Downsample action>> and set
 which you want to aggregate the original time series data.
 
 In this example, an ILM policy is configured for the `hot` phase. The downsample
-takes place after the initial index rollover, which for demonstration
-purposes is set to run after five minutes.
+takes place after the index is rolled over and the <<index-time-series-end-time, index time series end time>>
+has lapsed as the source index is still expected to receive writes until then.
+{ilm-cap} will not proceed with any action that expects the index to not receive
```

Comment:
receive major writes? After end_time is passed a backing index can still accept writes, but the expectation is that the volume of writes will be lower.